packages/component-library-react/package.json

```diff
@@ -135,7 +135,7 @@
     "@utrecht/badge-status-css": "workspace:*",
     "@utrecht/blockquote-css": "workspace:*",
     "@utrecht/breadcrumb-nav-css": "workspace:*",
-    "@utrecht/button-group-css": "workspace:*",
+    "@utrecht/button-group-react": "workspace:*",
```

Comment:
Je zult beide nodig hebben. Je kunt het onderstaande commando gebruiken om ook de root `pnpm-lock.yaml` bijgewerkt te krijgen.

```sh
pnpm --filter @utrecht/component-library-react add @utrecht/button-group-react@workspace:*
```

Reply:
Nee, dit is prima zo